formats/detailed_guide/frontend/examples/detailed_guide.json

```diff
@@ -0,0 +1,71 @@
+    "change_note": "The second entry in the table Examples of salary sacrifice has been amended to correct the explanation of how much of the salary is subject to tax and National Insurance contributions.",
+    "first_published_at": "2014-06-12T10:00:00+01:00"
+  },
+  "links": {
+    "lead_organisations": [
+      {
+        "content_id": "6667cce2-e809-4e21-ae09-cb0bdc1ddda3",
+        "title": "HM Revenue & Customs",
+        "base_path": "/government/organisations/hm-revenue-customs",
+        "api_url": "https://www.gov.uk/api/organisations/hm-revenue-customs",
+        "web_url": "https://www.gov.uk/government/organisations/hm-revenue-customs",
+        "locale": "en",
+        "analytics_identifier": "D25"
+      }
+    ],
+    "worldwide_organisations": [
+      {
+        "content_id": "6667cce2-e809-4e21-ae09-cb0bdc1ddda3",
+        "title": "HM Revenue & Customs",
+        "base_path": "/government/world/organisations/hm-revenue-customs",
+        "api_url": "https://www.gov.uk/api/content/government/world/organisations/hm-revenue-customs",
+        "web_url": "https://www.gov.uk/government/world/organisations/hm-revenue-customs",
+        "locale": "en",
+        "analytics_identifier": "D25"
+      }
+    ],
+    "policy_areas": [
+      {
+        "content_id": "8034be95-4ac2-4fff-93c5-e7514ed9504a",
+        "title": "Tax and revenue",
+        "base_path": "/government/topics/tax-and-revenue",
+        "api_url": "https://www.gov.uk/api/content/government/world/topics/tax-and-revenue",
+        "web_url": "https://www.gov.uk/government/world/topics/tax-and-revenue",
+        "locale": "en"
+      }
+    ],
+    "parent": [
+      {
+        "content_id": "6e1d83bc-d37a-447c-bdac-31be4b441417",
+        "title": "PAYE",
```

Comment:
This is a good point - it's showing the title of top-level-topic there.

I believe Whitehall can do this because content-api exposes the parent for any tag, so it displays that title of the `primary_specialist_sector`'s parent ("Business Tax") of the title of the actual `primary_specialist_sector` ("PAYE").

The eventual idea is to expand `links.parent` recursively with the dependency resolution, so that content-store has the full data to render this page.

cc @mobaig, who is converting the frontend of Whitehall to use the links hash (and thus parent).

Perhaps a chat tomorrow would be good?